core/src/components/badge/badge.tsx

```diff
@@ -27,10 +27,16 @@ export class Badge implements ComponentInterface {
   @Prop({ reflect: true }) color?: Color;
 
   /**
-   * Set to "xsmall" for even less height and width. Set to `"small"` for less height and width. Set to "medium" for slightly larger dimensions. Set to "large" for even greater height and width. Set to `"xlarge"` for the largest badge.
+   * Set to `"xxsmall"` for the smallest badge.
+   * Set to "xsmall" for even less height and width.
+   * Set to `"small"` for less height and width.
+   * Set to "medium" for slightly larger dimensions.
+   * Set to "large" for even greater height and width.
+   * Set to `"xlarge"` for the largest badge.
+   *
```

Comment:
```suggestion
   * Set to `"xxsmall"` for the smallest badge.
   * Set to "xsmall" for a very small badge.
   * Set to `"small"` for a small badge.
   * Set to "medium" for a medium badge.
   * Set to "large" for a large badge.
   * Set to `"xlarge"` for the largest badge.
   *
```

Should we do something like this here? It reads a bit confusing going from "the smallest" to "even less height" on the next size up.

Reply:
Works for me!